src/mlpack/tests/ksinit_test.cpp

```diff
@@ -230,8 +230,10 @@ TEST_CASE("IrisDataset", "[KSInitialization]")
 
   arma::mat dataset, labels;
 
-  data::Load("iris.csv", dataset, true);
-  data::Load("iris_labels.txt", labels, true);
+  if (!data::Load("iris.csv", dataset, true)) 
```

Comment:
```suggestion
  if (!data::Load("iris.csv", dataset)) 
```
Third argument not required.